backend/process_miner/logs_process_miner.py

```diff
@@ -0,0 +1,141 @@
+"""
+Module used for importing tagged logs into heuristic miner
+"""
+import os
+
+from pm4py.objects.log.adapters.pandas import csv_import_adapter
+
+from pm4py.objects.conversion.log import factory as conversion_factory
+
+from pm4py.algo.discovery.heuristics import algorithm as heuristics_miner
+from pm4py.visualization.heuristics_net import visualizer as hn_vis
+
+from pm4py.visualization.dfg import factory as dfg_vis_factory
+from pm4py.algo.discovery.dfg import factory as dfg_factory
+
+from pm4py.util import constants
+
+from pm4py.algo.filtering.pandas.attributes import attributes_filter
+
+POSSIBLE_APPROACHES = {"embedded": "embedded", "redirect": "redirect",
+                       "OAuth": "OAuth", "all": "all",
+                       "not available": "not available"}
+
+
+def create_dataframe():
+    """
+    create dataframe
+    """
+    dataframe = csv_import_adapter.import_dataframe_from_path(
+        'concated_files.csv', sep=",")
+    dataframe = dataframe.rename(
+        columns={'correlationId': 'case:concept:name',
+                 'timestamp': 'time:timestamp',
+                 'message': 'concept:name',
+                 'approach': 'case:approach'})
+    return dataframe
+
+
+def filter_by_approach(approach, dataframe):
+    """
+    dataframe gets filtered by approach
+    """
+    dataframe_approach = dataframe
+    if approach == POSSIBLE_APPROACHES['all']:
+        return dataframe_approach
+
+    all_approaches = attributes_filter.get_attribute_values(
+        dataframe, attribute_key="case:approach")
+    print("INFO: all approaches ", all_approaches)
```

Comment:
Consider using a logger instead of directly printing to the console. This makes managing the output a easier. (see `__main__.py` or other modules for example)